packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.styles.ts

```diff
@@ -368,5 +385,60 @@ export const getStyles = memoizeFunction((
 
   };

```

Comment:
I'm going to leave the const inside getStyles because a lot of the properties (colors for example) are defined based on the theme, which is a parameter to the function